server-spi/src/main/java/org/keycloak/utils/StringUtil.java

```diff
@@ -28,6 +28,10 @@ public static boolean  isNotBlank(String str) {
         return str != null && !"".equals(str.trim());
     }
 
+    public static boolean isNullOrEmpty(String str) {
```

Comment:
Are you adding because of the complexity of the `isBlank`?

Reply:
Wanted to keep the original behavior from the calling class + `isNotNull` is not the cheapest as it does a trim.